caffe2/operators/conv_op_cudnn.cc

```diff
@@ -54,7 +66,8 @@ class CudnnConvOpBase : public ConvPoolOpBase<CUDAContext> {
         deterministic_(
             OperatorBase::GetSingleArgument<int>("deterministic", 0)),
         cudnn_state_(OperatorBase::GetSingleArgument<int>("cudnn_state", 0)),
-        force_algo_(OperatorBase::GetRepeatedArgument<int>("force_algo", vector<int>{-1,-1,-1})) {
+        force_algo_(OperatorBase::GetRepeatedArgument<int>("force_algo", vector<int>{-1,-1,-1})),
+        enable_tensor_core_(OperatorBase::GetSingleArgument<bool>("enable_tensor_core", 0)) {
```

Comment:
My decision has always been that it should be opt-in (you're not the first person I've had this discussion with!), because it changes the numerics of the operation.

I've considered that behaviour across architectures should be as close as possible, so fp16 storage on Maxwell, Pascal and Volta should behave as close as they can (they're not identical already, but not too far from it). Enabling TensorCore by default changes that, admittedly very slightly, but I prefer all changes of that sort to be explicitly opt-in.